integration_tests/grid_fee.feature

```diff
@@ -16,6 +16,7 @@ Feature: GridFee integration tests
      |       5       |     60      |     10      |
      |       9       |     60      |     5       |
 
+  @slow
```

Comment:
Please use `@disabled` here, because I guess that these aren't suddenly slow but they are not working right?